crates/binstalk-downloader/src/resolver.rs

```diff
@@ -0,0 +1,64 @@
+use std::{net::SocketAddr, sync::Arc};
+
+use hyper::client::connect::dns::Name;
+use once_cell::sync::OnceCell;
+use reqwest::dns::{Addrs, Resolve};
+use trust_dns_resolver::config::{NameServerConfig, Protocol, ResolverConfig, ResolverOpts};
```

Comment:
Seems like I forgot to feature-gate this (given that I work on Windows, I failed to notice this because the lint warning only shows when compiling on Unix-like operating systems). This doesn't cause any problems, just a minor linter warning. I think it's a bit overkill to create a PR to fix just *this*. So, if you want, just make this small change, push, and it should be fixed :)
```suggestion
#[cfg(windows)]
use trust_dns_resolver::config::{NameServerConfig, Protocol, ResolverConfig, ResolverOpts};
```

Reply:
Thanks, I've opened #1365 fo this

Reply:
> not sure if you can make any changes while it's already in the merge queue?

You can cancel the merge and then update the PR, otherwise change to this PR will not be merged into main.